textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/fuzzytokenmatcher/FuzzyTokenMatcherTest.java

```diff
@@ -84,7 +84,7 @@ public void TestFuzzyTokenMatcherWithNoResults() throws Exception {
         ArrayList<Attribute> attributeList = new ArrayList<>();
         attributeList.add(TestConstants.DESCRIPTION_ATTR);
         List<ITuple> results = getQueryResults(query, threshold, attributeList, isSpanInformationAdded);
-        //Assert.assertEquals(0,results.size());
+        Assert.assertEquals(0,results.size());
     }
     
     @Test
```

Comment:
I have added one with 50 words and it works like a charm. Do we still need more ?